dev_scripts_helpers/github/dockerized_sync_gh_issue_labels.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+"""
+The script is designed to synchronize GitHub issue labels from a label
+inventory manifest file. It requires certain dependencies to be present (e.g.,
+`pygithub`) and thus it is executed within a Docker container.
+
+To use this script, you need to provide the input file, GitHub
```

Comment:
Let's put some examples of command lines, or better because of DRY let's say "The command lines are the same as sync_gh_...`